test/spec/fpd/enrichment_spec.js

```diff
@@ -310,6 +312,55 @@ describe('FPD enrichment', () => {
     });
   });
 
+  describe('privacy sandbox cookieDeprecationLabel', () => {
+    it('attempts to set ext.cdep on device obj when the gdprEnforcement module is not active', () => {
```

Comment:
The test does not match its description - and also mixes the behavior of enrichment and gdprEnforcement. IMO a better set of tests is:

- if `isActivityAllowed` is mocked to `true`, and the navigator API returns a promise to some label X, the enrichment puts X in `device.ext.cdep` (your tests only check that the navigator is invoked)
- if `isActivityAllowed` is mocked to `false`, the navigator API is not called and no enrichment happens
- if the navigator API returns a promise that rejects, the enrichment does not halt forever (I expect your current version to fail this one)
- if `validateRules` is passed `FIRST_PARTY_GVLID`, it will use `publisher.consents`  - this should be part of the gdprEnforcement suite